src/explore-education-statistics-admin/src/components/editable/ReorderableAccordion.tsx

```diff
@@ -82,21 +84,24 @@ const ReorderableAccordion = (props: ReorderableAccordionProps) => {
         <h2 className="govuk-heading-l govuk-!-margin-bottom-0">{heading}</h2>
 
         {sections.length > 1 &&
+          canUpdateRelease &&
           (!isReordering ? (
             <Button
               variant="secondary"
               className="govuk-!-font-size-16 govuk-!-margin-bottom-0"
               onClick={toggleReordering.on}
             >
-              Reorder<span className="govuk-visually-hidden"> data files</span>
+              Reorder<VisuallyHidden> data files</VisuallyHidden>
```

Comment:
Would create a `reorderHiddenText` prop for this text. Would suggest defaulting it to 'sections'.

Think you've done something like this in a previous PR so it's basically more of the same!